test/python/test_pack.py

```diff
@@ -353,19 +353,38 @@ def has_tail(path): return len(path.split('/', maxsplit=1)) > 1
 
 def validate_version_file(distribution_dir):
     original_keys = [
+        'TARANTOOL',
         project_name,
+        # default app dependencies
+        'luatest',
+        'cartridge',
+        # known cartridge dependencies
+        'membership',
+        'checks',
+        'vshard',
+        'http',
+        'frontend-core',
     ]
+
     if tarantool_enterprise_is_used():
-        original_keys.append('TARANTOOL')
         original_keys.append('TARANTOOL_SDK')
 
     version_filepath = os.path.join(distribution_dir, 'VERSION')
     assert os.path.exists(version_filepath)
 
+    version_file_content = {}
     with open(version_filepath, 'r') as version_file:
-        version_content = version_file.read()
-        for key in original_keys:
-            assert '{}='.format(key) in version_content
+        file_lines = version_file.read().strip().split('\n')
```

Comment:
may be `splitlines()`?

Reply:
If file ends with `\n\n`, `splitlines` will return `[..., '']` (last element is empty string).
I prefer using `strip + split('\n')`